PIL/Image.py

```diff
@@ -1129,7 +1129,8 @@ def _crop(self, im, box):
 
     def draft(self, mode, size):
         # type: (Mode, Size) -> Any
-        # TODO: return type clarification
+        # FIXME TYPING: return type clarification
+        # FIXME TYPING: thumbnail calls as (None, size), so is it Optional[Mode]?
```

Comment:
Yes, optional mode. The JPEG one is the only one I see, and it returns self, which is not really consistent with the rest of the API.